src/database/volatile/mod.rs

```diff
@@ -5,9 +5,10 @@
 //!
 //! #### Authorship
 //! - Max Fierro, 2/24/2024 (maxfierro@berkeley.edu)
+//! - Implementation: Casey Stanford, 4/10/2024 (cqstanford@berkeley.edu)
```

Comment:
You can delete my name if you want, but let's keep the author list homogenous (w/o the "Implementation:") -- everyone can see who did what using git blame regardless